_includes/desktop-update.md

```diff
@@ -0,0 +1,11 @@
+<!-- This text will be included in the Mac and Windows Install topic -->
+
+Starting with Docker Desktop 3.0.0, updates to Docker Desktop will be available automatically as delta updates from the previous version. When an update is available, Docker Desktop displays an icon to indicate the availability of a newer version. You can choose when to start the download and installation process.
+
+> To encourage developers to stay up to date, Docker Desktop displays a reminder two weeks after an update becomes available. You can dismiss this daily reminder using the **Snooze** option.
```

Comment:
An option sounds a bit like a setting to me, maybe some users will start asking where to find this option in the settings?

Reply:
@mat007 Updated it to 'clicking' Snooze. Also, there's a screenshot that displays this option. 

![image](https://user-images.githubusercontent.com/47779042/117157612-a17e6380-adb6-11eb-834a-a4481900c2b0.png)